catalog/core/tests/test_views.py

```diff
@@ -55,28 +67,48 @@ def test_logout(self):
 
 class ProfileViewTest(BaseTest):
     def test_profile_view(self):
-        self.without_login_and_with_login_test('core:user_profile')
+        self.without_login_and_with_login_test(USER_PROFILE_URL)
 
+    @settings(max_examples=30, perform_health_check=False)
     def test_profile_update(self):
         url = self.reverse(USER_PROFILE_URL, query_parameters={'format': 'json'})
         self.login()
+        user_info = models(User).example()
```

Comment:
This should be in the given decorator as GENERATE_USER. See the comment below for detail.